research/object_detection/colab_tutorials/object_detection_tutorial.ipynb

```diff
@@ -158,23 +159,12 @@
       "outputs": [],
       "source": [
         "%%bash\n",
-        "cd models/research/\n",
-        "protoc object_detection/protos/*.proto --python_out=."
-      ]
-    },
-    {
-      "cell_type": "code",
-      "execution_count": null,
-      "metadata": {
-        "colab": {},
-        "colab_type": "code",
-        "id": "s62yJyQUcYbp"
-      },
-      "outputs": [],
-      "source": [
-        "%%bash \n",
         "cd models/research\n",
-        "pip install ."
+        "# Compile protos.\n",
```

Comment:
I think this will be confusing because we already point to the installation instructions above. So my suggestion would be to just fix the link to the installation instructions (it's currently broken).

Reply:
You're right, I just updated the link. But more over I'm a bit confused why it even mentions to follow the install instructions. Colab requires all the steps in the installation instructions to be performed. What I would really expect is something that reads "if you are working locally **and you have already installed TF object detection, skip these install cells"